src/components/common/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx

```diff
@@ -137,17 +98,16 @@ Props) => {
 
   const [userHasPermission, onlyForceAction] = useDialogActionPermissions(
     colony,
-    false, // isVotingExtensionEnabled,
+    isVotingReputationEnabled,
     requiredRoles,
-    [domainId],
+    [values.fromDomain],
   );
 
-  // const cannotCreateMotion =
-  //   votingExtensionVersion ===
-  //     VotingReputationExtensionVersion.FuchsiaLightweightSpaceship &&
-  //   !values.forceAction;
+  const cannotCreateMotion =
+    votingReputationVersion === noMotionsVotingReputationVersion &&
+    !values.forceAction;
 
-  const canMakePayment = userHasPermission; // && isOneTxPaymentExtensionEnabled;
+  const canMakePayment = userHasPermission && isOneTxPaymentEnabled;
 
   const inputDisabled = !canMakePayment || onlyForceAction || isSubmitting;
```

Comment:
Similar thought about wrapping up as much logic as possible so we're only exposing what the view needs (and reducing the number of lines someone needs to read through before they get to the view itself)